rfc.sh

```diff
@@ -5,19 +5,19 @@
 # we have to run the command in an if-statement.
 
 UPSTREAM=${GLUSTER_UPSTREAM}
-if [ "x$UPSTREAM" -eq "x" ]; then
+if [ "x$UPSTREAM" = "x" ]; then
```

Comment:
This should be "=="

Reply:
amar@bulde:~/work/gluster/glusterfs$ DRY_RUN=1 ./rfc.sh -f
./rfc.sh: 8: [: x: unexpected operator
./rfc.sh: 20: [: xorigin: unexpected operator

(this is when I change it to `==`, had to use `=` only!

Reply:
Sorry, it seems '=' is the right POSIX method.